utils/update_checkout/update-checkout-config.json

```diff
@@ -93,6 +93,8 @@
                 "swift-tools-support-core": "main",
                 "swiftpm": "main",
                 "swift-argument-parser": "1.0.1",
+                "swift-atomics": "1.0.2",
+                "swift-collections": "1.0.1",
```

Comment:
Something's changed upstream that made these projects required now, their lack causes a build breakage.